src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,83 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String SUPPLY_VALUE = "supply";
+    private static final String BUY_VALUE = "buy";
+    private static final String RESULT_VALUE = "result";
+    private static final String COMMA_VALUE = ",";
+    private static final String WHITESPACE = System.lineSeparator();
```

Comment:
is it `WHITESPACE` or it is lineSeparator? 
because we have a whitespace when we press `space` and we have line separator when we press `enter`